AudioLink/Scripts/AudioLinkController.cs

```diff
@@ -64,7 +67,21 @@ void Update()
 
             void Start()
             {
-                if (audioLink == null) Debug.Log("Controller not connected to AudioLink");
+                if (audioLink == null) {
+                    Debug.LogError("Controller not connected to AudioLink");
+                    return;
+                }
+                if (themeColorController == null) {
+                    // This is here in case someone upgraded AudioLink, which updates
+                    // everything in the prefab, but not the outermost properties of the prefab.
+                    // TODO: Double check that this is how upgrading ends up working.
```

Comment:
Did you verify this yourself yet, or shall I do it?

Reply:
I have not yet verified this. I'd appreciate it if you have time